spec/index.bs

```diff
@@ -1179,10 +1163,19 @@ partial interface HTMLScriptElement {
 
 This document modifies {{HTMLScriptElement}}s. Each script has:
 
-: an associated string <dfn export for="HTMLScriptElement">script text</dfn>.
-::  A string, containing the body of the script to execute that was set
-    through a compliant sink. Equivalent to script's
-    [=child text content=]. Initially an empty string.
+: an associated boolean <dfn export for="HTMLScriptElement">is trusted</dfn>.
```

Comment:
I think `for` is either comma or vertical bar separated so you can define it for HTML and SVG `script` once.